src/Action.ts

```diff
@@ -10,10 +10,10 @@ export class Action {
     let filtered = pullRequests.filter((pr) => pr.user?.login == this.userName)
 
     if (filtered.length > 1){
-      let newest = filtered[0].head.ref
```

Comment:
I think we still need this guy - we want to tell the closed PR to look at the newest one.